<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessBoard</class>
 <widget class="QDialog" name="ChessBoard">
  <property name="windowTitle">
   <string>Шахматы :: Доска</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="rowCount">
      <number>10</number>
     </property>
     <property name="columnCount">
      <number>10</number>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="horizontalHeaderVisible">
      <bool>false</bool>
     </property>
     <property name="verticalHeaderVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Запись в файл</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: lightgreen;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Выход</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: lightcoral;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
